recipe/meta.yaml

```diff
@@ -1,5 +1,13 @@
 {% set name = "qiskit-aer" %}
 {% set version = "0.13.1" %}
+{% set build = 1 %}
```

Comment:
FYI. I noticed we made a mistake here (not bumping the build number). Because we altered the build string in this PR, during uploading conda-build considered the CPU builds as distinct, and the new builds got uploaded. Since we're rebuilding in other PRs, this will be corrected.

Reply:
I thought the changes here added 200 to the CPU build number to keep it higher than the GPU build and that we did want it uploaded with that new number?

Reply:
Yeah you're right, I forgot we switched to prioritize the cpu builds...